csrc/attention/attention_kernels.cu

```diff
@@ -300,9 +309,7 @@ __device__ void paged_attention_kernel(
   constexpr int V_VEC_SIZE = MIN(16 / sizeof(scalar_t), BLOCK_SIZE);
   using V_vec = typename Vec<scalar_t, V_VEC_SIZE>::Type;
   using L_vec = typename Vec<scalar_t, V_VEC_SIZE>::Type;
-#ifdef ENABLE_FP8_E5M2
-  using V_quant_vec = typename Vec<cache_t, V_VEC_SIZE>::Type;
-#endif
+  using V_quant_vec = typename Vec<cache_t, V_VEC_SIZE>::Type; 
```

Comment:
When `ENABLE_FP8_E5M2` is not defined in `setup.py`, removing the macro `ENABLE_FP8_E5M2` here will cause compile error : `attention_kernels.cu(312): error: class "vllm::Vec<uint8_t, 4>" has no member "Type"`.
```suggestion
  using V_quant_vec = typename Vec<cache_t, V_VEC_SIZE>::Type; 
```

Reply:
This is an incompatibility issue. `int8 kv cache` and `fp8 kv cache` both need this `V_dequant_vec`, but `int8 kv cache` doesn't need the macro `ENABLE_FP8_E5M2`. I wonder if we can define two types, like `V_int8_vec` and `V_fp8_vec`. Or any suggestions for this issue?

Reply:
Define two different types is also an option. I emailed you a git patch to fix the incompatibility. You could apply it.

Reply:
Hello, I just checked my email, it seemed that I didn't receive the git patch. Could you please tell me which email address you send it to and email the git patch again?

Reply:
Send to 007aabbcc411@gmail.com again. Please check it.

Reply:
Your git patch can fix this issue. Thanks!